src/Components/Components/ref/Microsoft.AspNetCore.Components.netstandard2.0.cs

```diff
@@ -566,6 +534,13 @@ public abstract partial class UriHelperBase : Microsoft.AspNetCore.Components.IU
         protected void TriggerOnLocationChanged(bool isinterceptedLink) { }
     }
 }
+namespace Microsoft.AspNetCore.Components.CompilerServices
+{
+    public static partial class RuntimeHelpers
```

Comment:
I can remove the original copy of this after updating the compiler. It's easiest to do this as a three-step operation.